VAMobile/e2e/tests/Appointments.e2e.ts

```diff
@@ -168,7 +168,6 @@ describe('Appointments Screen', () => {
   it('should tap on and show past appointments', async () => {
     await element(by.id('appointmentsTestID')).scrollTo('top')
     await element(by.text('Past')).tap()
-    await expect(element(by.id(Appointmentse2eConstants.PAST_APPOINTMENT_1_ID))).toExist()
```

Comment:
You can also remove the `PAST_APPOINTMENT_1_ID` const since it's no longer being used

Reply:
removed